.travis/update-version.sh

```diff
@@ -26,7 +26,7 @@ commit_files() {
     git checkout -b ${NEW_VERSION}
 
     # Stage the modified files
-    git add ./**/pom.xml
+    git add ./pom.xml ./*/pom.xml ./*/*/pom.xml
```

Comment:
We can just do a `git add -A`. We are only changing the pom files in this script and we still have to manually merge the PR

Reply:
I thought that we wanted to explicitly limit it to just `pom.xml` files (I guess another safeguard?). If that is not the case then yeah lets just do `git add .` / `git add -A`

Reply:
Yea let's just add all since we have to merge a PR manually still